src/Microsoft.IdentityModel.Abstractions/ITelemetryClient.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.IdentityModel.Abstractions
+{
+    /// <summary>
+    /// Interface for Telemetry tracking.
+    /// </summary>
+    public interface ITelemetryClient
+    {
+        /// <summary>
+        /// Gets or sets the application or client ID that telemetry is being sent for.
+        /// </summary>
+        Guid ClientId { get; set; }
+
+        /// <summary>
+        /// Perform any necessary bootstrapping for the telemetry client.
+        /// </summary>
+        /// <remarks>
+        /// The expectation is that this should only be called once for the lifetime of the object however the
+        /// implementation should be idempotent.
+        /// </remarks>
+        void Initialize();
```

Comment:
Always a good idea to use modifiers such as 'internal, pubilc, ...'

Reply:
for <C#8.0, you can't set access modifiers on interface methods. Is that what the comment was talking about or something else?